twisted/web/test/test_http2.py

```diff
@@ -1971,6 +1972,30 @@ def validate(streamID):
         return a._streamCleanupCallbacks[1].addCallback(validate)
 
 
+    def test_unnecessaryWindowUpdateForStream(self):
+        """
+        When a WindowUpdate frame is received for a stream but no data is
+        currently waiting, nothing exciting happens.
+        """
+        f = FrameFactory()
+        transport = StringTransport()
+        conn = H2Connection()
+        conn.requestFactory = DummyHTTPHandler
+
+        # Send the request.
+        frames = buildRequestFrames(
+            self.postRequestHeaders, self.postRequestData, f
+        )
+        frames.insert(1, f.buildWindowUpdateFrame(streamID=1, increment=5))
```

Comment:
And with the above code, I think that the current big  comment block can just go away :)